packages/core/forms/src/components/fields/FieldInput.vue

```diff
@@ -17,11 +17,27 @@
       :placeholder="schema.placeholder"
       :readonly="schema.readonly"
       :required="schema.required"
-      :type="inputType"
+      :type="displayInputType"
       :width="schema.width"
       @blur="onBlur"
       @update:model-value="onInput"
-    />
+    >
+      <template
+        v-if="schema.inputType === 'password'"
+        #after
+      >
+        <VisibilityOffIcon
+          v-if="eyeOpen"
+          role="button"
+          @click="() => { eyeOpen = !eyeOpen }"
+        />
+        <VisibilityIcon
+          v-else
+          role="button"
+          @click="() => { eyeOpen = !eyeOpen }"
+        />
+      </template>
+    </KInput>
```

Comment:
I think this feature should be built-in in Kongponents. Let's implement as-is and I'll propose it to the design system.